spec/lib/data_update_scripts/append_collective_noun_to_community_name_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+require Rails.root.join(
+  "lib/data_update_scripts/20201228194641_append_collective_noun_to_community_name.rb",
+)
+
+describe DataUpdateScripts::AppendCollectiveNounToCommunityName do
+  context "when collective_noun_disabled is false" do
+    it "appends the collective_noun to the community_name" do
+      described_class.new.run
+      expect(SiteConfig.collective_noun_disabled).to eq(false)
+      expect(SiteConfig.community_name).to eq("DEV(local) Community")
```

Comment:
This continues to return `"DEV(local)"` rather than `"DEV(local) Community"` and after dropping into `pry` and trying a bunch of different things, I still cannot get this to work properly. It almost feels like I need a `reload` here, but I'm unable to use that on a `string` -- thoughts? Am I missing something? 😅 

cc @vaidehijoshi and @maestromac